### PR TITLE
Add health status to components endpoint

### LINKED ISSUE #1599: The /client.config endpoint must query the services health endpoint
We must query the health endpoint for each service returned on the controller api call to check the status

### PR DESCRIPTION
**Pull request by chrismatix**
fixes #1599

### CODE REVIEW

File: lib/java/spring/kafka/healthcheck/src/main/java/co/airy/spring/kafka/healthcheck/ProducerHealthCheck.java [hunk 1]
```diff
@@ -1,37 +0,0 @@
...
-import org.apache.kafka.clients.producer.ProducerRecord;
-import org.springframework.stereotype.Service;
-
-import java.time.Instant;
-
-@Service
-public class ProducerHealthCheck {
-    private final KafkaProducer<String, HealthCheck> producer;
-
-    ProducerHealthCheck(KafkaProducer<String, HealthCheck> producer) {
```
Comment (chrismatix) [inline lib/java/spring/kafka/healthcheck/src/main/java/co/airy/spring/kafka/healthcheck/ProducerHealthCheck.java:15]: This wasn't actually used so I removed it

File: infrastructure/helm-chart/charts/apps/charts/api/charts/api-admin/templates/deployment.yaml [hunk 2]
```diff
@@ -51,8 +51,6 @@ spec:
             configMapKeyRef:
               name: kafka-config
               key: KAFKA_COMMIT_INTERVAL_MS
-        - name: SERVICE_NAME
```
Comment (pascal-airy) [inline infrastructure/helm-chart/charts/apps/charts/api/charts/api-admin/templates/deployment.yaml:54]: Why did you remove the service names?

Comment (chrismatix) [reply infrastructure/helm-chart/charts/apps/charts/api/charts/api-admin/templates/deployment.yaml:54]: @pascal-airy Because I don't think that they are used. There used to be an implicit usage in the `ProducerHealthCheck`, but that was also unused so I removed both.

**Review by pascal-airy — APPROVED** [review]
lgtm!
